### PR TITLE
Uniform Logger

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

KEB code base uses multiple loggers. The following PR changes logger invocations to either slog or Loggrus that log messages as json messages (with custom handler set in the PR) and details timestamp (default in slog and customised in the PR for Loggrus).

Changes proposed in this pull request:

- corrected order of Logrus initialisation in main methods,
- removed `log` package usages,
- set missing json handler for slog and default logrus loggers.

**Related issue(s)**

### CODE REVIEW

File: cmd/broker/main.go [hunk 1]
```diff
@@ -228,10 +227,24 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// set default formatted
+	logrus.SetFormatter(&logrus.JSONFormatter{
+		TimestampFormat: time.RFC3339Nano,
+	})
+	logs := logrus.New()
+	logs.SetFormatter(&logrus.JSONFormatter{
```
Comment (jaroslaw-pieszka) [inline cmd/broker/main.go:235]: Code duplication. Why this is necessary to repeat the sentence. If we SetFormatter for `logrus` and then create new `log` would it not be set to json already?

File: cmd/broker/main.go [hunk 2]
```diff
@@ -314,33 +318,33 @@ func main() {
 		kebConfig.NewConfigMapConverter())
 	componentsProvider := runtime.NewComponentsProvider()
 	gardenerClusterConfig, err := gardener.NewGardenerClusterConfig(cfg.Gardener.KubeconfigPath)
-	fatalOnError(err)
+	fatalOnError(err, logs)
```
Comment (jaroslaw-pieszka) [inline cmd/broker/main.go:321]: Is this the only way, to pass always logger to this function? No way to use some `global` logger?

File: cmd/broker/main.go (hunk 1 above)
Comment (ralikio) [reply cmd/broker/main.go:235]: In the implementation there is one global instance of logrus that we need to configure for all the places where explicit logrus instance has not been created. Additionally, `logrus.New` does not seem to copy configuration from that global instance when creating a new one. That is why duplicated configuration is required.

File: cmd/broker/main.go (hunk 2 above)
Comment (ralikio) [reply cmd/broker/main.go:321]: I did it as a safe guard to use logger that is passed to methods in which `fataOnError` is invoked in. This way we are keeping all the labels that fataOnError should log.

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**